liferay-content-management/liferay-portal
# LPD-23683 Release FF - Removal: Improvements for Mine and Recent filter in WC

### What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-23683

Delete release feature Flag LPD-11218
 
### How am I fixing it?

Deleting feature flag

### How can you verify that it works?

Disabling or deleting all feature flags that you have in your bundle and going to content management and check if now we have two new filters: now and recent. Also you have to check that these requirement are working as expected:

- The “Mine” filters should include all the content that has been created by me (the user that is filtering). So that I am the author of the first version.
- The results of the “Mine” filter should include information of the date when it was created.
- The “Recent” filter should show all the content created and edited by me, and ordered by date of these interactions.
